BaseApp/COD3000.TXT

```diff
@@ -18,99 +18,132 @@ OBJECT Codeunit 3000 DotNet_Array
       DotNetArray@1000 : DotNet "'mscorlib'.System.Array";
 
     [External]
+    [LineStart(2)]
```

Comment:
Be careful when you commit these lines.
`LineStart` should not go to the codebase 😃